### PR TITLE
STYLE: apply section on grants page

### PR DESCRIPTION
**Pull request by barbaraperic**
## Changes
- updated the Apply now section in the grants page

## Visuals

Screenshot 2024-04-19 at 09 48 18
Screenshot 2024-04-19 at 09 48 12

### CODE REVIEW

File: src/app/grants/page.tsx [hunk 1]
```diff
@@ -171,16 +172,25 @@ export default function Grants() {
...
-        <TextLink href={FILECOIN_URLS.social.slack.href}>Learn More</TextLink>
-        <GetInvolvedList />
+        <Button
+          className="sm:self-start"
+          variant="primary"
+          href={FILECOIN_URLS.grants.email}
+        >
+          Apply Now
+        </Button>
       </section>
```
Comment (mirhamasala) [inline src/app/grants/page.tsx:194]: @barbaraperic Can you please have a look at the Figma designs, and confirm that we'll need to this component in a few pages? If so, let's create a component for it!

Comment (barbaraperic) [reply src/app/grants/page.tsx]: You're right, there is a similar section. I've extracted the section styles and heading, left paragraph and button as children as they can vary with icons and inner text links.

Screenshot 2024-04-22 at 09 51 18

File: src/app/_components/CallToActionSection.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Heading } from '@/components/Heading'
+
+type CallToActionSectionProps = {
+  title: string
+  children?: React.ReactNode
+}
+
+export function CallToActionSection({
```
Comment (mirhamasala) [inline src/app/_components/CallToActionSection.tsx:8]:
```suggestion
export function CTASection({
```

File: src/app/grants/page.tsx (hunk 1 above)
Comment (mirhamasala) [reply src/app/grants/page.tsx]: I understand the thinking. Still, we should try and move the text and button inside of the component. Not unlike we do for the optional `cta` in  `Card` (please note that `Card` needs some refactoring so asses yourself what can be reused!)

```
        {cta && (
          <CustomLink
            href={cta.href}
            aria-label={cta.ariaLabel}
            className="absolute inset-0 rounded-lg focus:outline-2 focus:outline-brand-100"
          >
            <span className="absolute bottom-4 left-4 inline-flex items-center gap-2 text-brand-300">
              {cta.icon || <BookOpen size={24} aria-hidden={true} />}
              <span>{cta.text || 'Learn More'}</span>
            </span>
          </CustomLink>
        )}
```        

As for the up arrow icon with outgoing links we should add that as the permanent icon under `Button` for external links.

Comment (mirhamasala) [reply src/app/grants/page.tsx]: Also - You don't need to take all options into account in this PR - Make the component work for this PR - and then we can get to the variants once we need to.

**Pull request merged**